src/storage/hummock_test/src/lib.rs

```diff
@@ -28,7 +28,6 @@ mod local_version_manager_tests;
 mod snapshot_tests;
 #[cfg(test)]
 mod state_store_tests;
-#[cfg(any(test, feature = "test"))]
```

Comment:
any context about removing `#[cfg(any(test, feature = "test"))]` ? Instead, shall we need to add `#[cfg(test)]` ?

Reply:
test_utils is invisible to other crates with this flag. Adding `#[cfg(test)]` won't help. Another alternative is to keep `#[cfg(any(test, feature = "test"))]` but specify `features = ["test"]` when introducing risingwave_hummock_test in stream/Cargo.toml. Both looks okay to me.